service/awsconfig/v6/resource/cloudformation/adapter/auto_scaling_group.go

```diff
@@ -41,5 +41,9 @@ func workerCountRatio(workers int, ratio float32) string {
 	value := float32(workers) * ratio
 	rounded := int(value + 0.5)
 
+	if workers > 0 && rounded == 0 {
+		rounded = 1
+	}
```

Comment:
v6 got just frozen. We need to introduce a new version bundle.

Reply:
For me it makes sense to include this in v6 since its also a bug fix.

Reply:
Oh so it is not yet frozen?

Reply:
We should then also add another changelog. 

When do you plan to freeze this here then?

Reply:
I'll add to the changelog. This will be the final change.